frontend/src/app/components/ApprovalModal.tsx

```diff
@@ -1,41 +1,30 @@
-import React from 'react';
+import { Button } from "@nextui-org/button";
+import React from "react";
+import WatchIcon from "./svgs/WatchIcon";
 
-const ApprovalModal = () => {
-  return (
-    <section className="flex flex-col items-center justify-center max-w-screen-sm px-6 py-12 mx-auto my-8 bg-white rounded-3xl shadow-lg">
-    <HeaderImage
-        src="/images/close.svg"
-        alt="Notification Icon"
-        additionalClasses="w-6 h-6 relative cursor-pointer ml-auto -mt-6 -mr-1"
-    />
-      <HeaderImage
-        src="/images/watch.svg"
-        alt="Approval Pending"
-        additionalClasses="w-12 h-12 mt-7"
-      />
-      <h1 className="mt-4 text-xl font-bold text-center">Waiting for Approval</h1>
-      <p className="mt-8 text-center">
-        Thank you for posting your listing! It is currently under review to ensure quality and compliance.
-        <br />
-        <br />
-        Please check back later for updates on its status. We appreciate your patience!
-      </p>
-      <PreviewButton />
-    </section>
-  );
-};
-
-const HeaderImage = ({ src, alt, additionalClasses }: { src: string, alt: string, additionalClasses: string }) => (
-    <img loading="lazy" src={src} alt={alt} className={`mx-auto ${additionalClasses}`} />
-);
+const ApprovalModal: React.FC = () => {
+    return (
+        <section className="mx-auto my-8 flex max-w-screen-sm flex-col items-center justify-center rounded-3xl bg-white px-6 py-12 shadow-lg">
```

Comment:
Is it possible to use the modal component from NextUI?

Reply:
This is still work in progress for @rianfrancis

Reply:
noted